wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/WbGetEntitiesAction.java

```diff
@@ -236,7 +236,14 @@ public WbGetEntitiesAction(ApiConnection connection, String siteIri) {
 					EntityIdValue entityIdValue = EntityIdValueImpl.fromId(entityNode.get("id").asText(), siteIri);
 					if(entityIdValue instanceof MediaInfoIdValue) {
 						//TODO: bad hack, it would be much nicer if the API would return the page title
```

Comment:
I thought the title was returned with `&props=info`?
https://www.wikidata.org/w/api.php?action=wbgetentities&ids=Q42&props=info

Reply:
Or was it something like this you were asking about?
https://www.wikidata.org/w/api.php?action=wbgetentities&ids=Q42&props=sitelinks&sitefilter=enwiki

Reply:
It does not seem to work for Commons at least!

Reply:
The `sitelinks` does however: https://commons.wikimedia.org/w/api.php?action=wbgetentities&ids=Q42&props=sitelinks&sitefilter=enwiki

And the `title` which is just a Special page:  https://commons.wikimedia.org/w/api.php?action=wbgetentities&ids=Q42&props=info
```json
{
    "entities": {
        "Q42": {
            "pageid": 0,
            "ns": -1,
            "title": "d:Special:EntityPage/Q42",
            "lastrevid": 1563235623,
            "modified": "2022-01-16T12:32:28Z",
            "type": "item",
            "id": "Q42"
        }
    },
    "success": 1
}
```

Reply:
Well that id `M114395392`  is totally missing and not indexed in any of the Common namespaces as seen:  https://commons.wikimedia.org/w/index.php?search=M114395392&title=Special:Search&profile=advanced&fulltext=1&ns0=1&ns1=1&ns2=1&ns3=1&ns4=1&ns5=1&ns6=1&ns7=1&ns8=1&ns9=1&ns10=1&ns11=1&ns12=1&ns13=1&ns14=1&ns15=1&ns100=1&ns101=1&ns102=1&ns103=1&ns104=1&ns105=1&ns106=1&ns107=1&ns460=1&ns461=1&ns486=1&ns487=1&ns490=1&ns491=1&ns828=1&ns829=1&ns1198=1&ns1199=1&ns2300=1&ns2301=1&ns2302=1&ns2303=1

So definitely "missing" :-)

Reply:
@wetneb Figured it out !  ... You have to get the Page info basically directly via ?action=query ....  M-id = (M + pageid)

https://commons.wikimedia.org/w/api.php?action=query&pageids=114395392&format=json&formatversion=2

https://www.mediawiki.org/wiki/API:Pageprops
So ... 
https://commons.wikimedia.org/w/api.php?action=query&pageids=114395392&format=json&formatversion=2&prop=pageprops

https://commons.wikimedia.org/w/index.php?title=File:Hieroglyphica_1602_(7885430).jpg&action=info

Maybe this page could be updated with some of that info about Page details:
https://commons.wikimedia.org/wiki/Commons:Structured_data/Get_involved/Finding_data#Wikimedia_Commons_Query_Service